wpeverest/user-registration
# Enhance - Copy icon on shortcode column in registration forms list table

### All Submissions:






### Changes proposed in this Pull Request:
This PR adds a new shortcode copy icon in the shortcode column of the user registration form's table list.

### How to test the changes in this Pull Request:

1. Navigate to the All Forms page and observe the registration form's list table.
2. There you will see a new copy shortcode icon, when it gets clicked then the shortcode will be copied to clipboard.

![copy-shortcode](https://user-images.githubusercontent.com/23380477/89612175-e1073600-d89e-11ea-89f3-23216e4e3495.PNG)


### Types of changes:




### Other information:




### Changelog entry

> Enhance - Copy icon on shortcode column in registration forms list table.
